Formula/luajit.rb

```diff
@@ -81,16 +82,16 @@ def install
 --- a/src/lj_alloc.c
 +++ b/src/lj_alloc.c
 @@ -72,13 +72,56 @@
- 
+
  #define IS_DIRECT_BIT		(SIZE_T_ONE)
- 
+
```

Comment:
I think the extra spaces in the `luajit` patch should be removed. They were probably automatically removed by your editor.

Reply:
yeah, I will put it back. good call